requirements.txt

```diff
@@ -0,0 +1,13 @@
+awscli==1.22.55
+numpy==1.18.4
+biopython==1.78
+ipywidgets
+jupyter_contrib_nbextensions
+nbconvert==6.5.0
+openpyxl==3.0.3
+xlrd==1.2.0
+statsmodels==0.11.1
+WeasyPrint==54.2
+onecodex[all,reports]==0.10.0
```

Comment:
`all` doesn't include `reports`?

Reply:
I'm not sure, I just copied this line over 😁 I'll test it out and remove the reports param in a subsequent PR if it's redundant